### PR TITLE
Shop tooltip adaptions for standard products

### PR DESCRIPTION
**Pull request by gjn**
@rfrancel I've created this PR to merge it to be able to deploy to integration.

### CODE REVIEW

File: chsdi/models/vector/stopo.py [hunk 1]
```diff
@@ -2439,12 +2418,32 @@ class DHM25Perimeter(Base, Vector):
 register_perimeter('ch.swisstopo.digitales-hoehenmodell_25_reliefschattierung', DHM25Perimeter)
 
 
-class SwissBuildings3dPerimeter(Base, Vector):
+class ShopStandardClass:
+    __template__ = 'templates/htmlpopup/shop_standard.mako'
+    title_de = Column('s_title_de', Text)
+    title_fr = Column('s_title_fr', Text)
+    title_it = Column('s_title_it', Text)
+    title_en = Column('s_title_en', Text)
```
Comment (gjn) [inline chsdi/models/vector/stopo.py]: We have this twice.

**Comment by gjn** [comment]
Some tests are failing because db does not seem to be ready for all layers. Let's discuss this tomorrow.

Thanks alot for your efforts!

**Comment by rfrancel** [comment]
I'm deploying the branch now

**Comment by gjn** [comment]
Tremendous effort. Thanks alot @oterral @AFoletti and @oterral 

I'll send out the validation mail. Let's keep it a little stable.

**Comment by rfrancel** [comment]
here is a test link with all teh layers going to prod tomorrow:
[test link](https://mf-geoadmin3.dev.bgdi.ch/?api_url=frr_shop_tooltips&topic=ech&lang=fr&X=190000.00&Y=660000.00&zoom=1&bgLayer=ch.swisstopo.swissimage&layers=ch.swisstopo.swissbuildings3d,ch.swisstopo.pixelkarte-farbe-pk25.noscale,ch.swisstopo.pixelkarte-farbe-pk50.noscale,ch.swisstopo.pixelkarte-farbe-pk100.noscale,ch.swisstopo.pixelkarte-farbe-pk200.noscale,ch.swisstopo.swissimage-product,ch.swisstopo.swisstlm3d-karte-farbe,ch.swisstopo.swissalti3d-reliefschattierung&layers_visibility=true,false,false,false,false,false,false,false)

**Comment by rfrancel** [comment]
the images were not displayed anymore in papier and fertig produkte. I corrected that.

File: chsdi/locale/empty_chsdi.po [hunk 2]
```diff
@@ -1512,9 +1518,19 @@ msgstr ""
 msgid "ch.swisstopo.lk25-papierkarte.metadata"
 msgstr ""
 
-msgid "ch.swisstopo.lk25-papierkarte.metadata.name"
+msgid "ch.swisstopo.lk25-papierkarte.metadata.name_de"
+msgstr ""
+
+msgid "ch.swisstopo.lk25-papierkarte.metadata.namefr"
```
Comment (gjn) [inline chsdi/locale/empty_chsdi.po]: typo?

**Pull request merged**